udata_front/theme/gouvfr/assets/js/config.ts

```diff
@@ -85,6 +85,8 @@ export const unchecked_types = _jsonMeta("unchecked-types");
 export const api_root = _meta("api-root");
 export const api_2_root = _meta("api-2-root");
 
+export const metrics_api_root = _meta("metrics-api");
```

Comment:
I wish we would not need to query metrics api in admin. We should probably add it as dataset metrics instead.

Reply:
soon 👀

Reply:
Could we remove the current metrics fetch logic? And use `dataset.metrics.downloads` that could default to 0 for starting and be replaced by real values when computed?